### PR TITLE
feat(editor): Don't ingest external images (SDNTB-231)

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by Aca-jov** [comment]
:+1:

File: scripts/superdesk/editor2/editor.js [hunk 1]
```diff
@@ -562,6 +562,13 @@ function SdTextEditorBlockEmbedController($timeout, editor, renditions) {
                 vm.onBlockChange();
             });
         },
+        isEditable: function(picture) {
+            if (picture._type === 'externalsource') {
```
Comment (petrjasek) [inline scripts/superdesk/editor2/editor.js]: this will be set via specific search provider?

Comment (ioanpocol) [inline scripts/superdesk/editor2/editor.js]: All search providers should set this because there is a logic on frontend for it.

**Comment by akintolga** [comment]
:+1:

File: scripts/superdesk/editor2/editor.js [hunk 2]
```diff
@@ -562,6 +562,13 @@ function SdTextEditorBlockEmbedController($timeout, editor, renditions) {
...
+                return false;
+            } else {
+                return true;
+            }
```
Comment (gbbr) [inline scripts/superdesk/editor2/editor.js]: You can replace all of this function with `return picture._type === 'externalresource'`

**Comment by gbbr** [comment]
👍

**Pull request merged**